packages/corvid-cli/src/apps/clone.js

```diff
@@ -61,8 +61,9 @@ async function extractMetasiteIdAndName(url, cookie) {
     const site = siteList.find(
       site =>
         site.publicUrl &&
-        normalize(site.publicUrl, { forceHttps: true }) ===
-          publicSiteOrEditorUrl
+        publicSiteOrEditorUrl.startsWith(
```

Comment:
what if i have more than 1 site that starts with the given string ?

Reply:
you mean we need to add at least '/' at end?